### PR TITLE
feat: Secretsmanager secret rotation for master user password

### PR DESCRIPTION
**Pull request by magreenbaum**
## Description
Add support for managed master user password rotation. 

## Motivation and Context
https://docs.aws.amazon.com/secretsmanager/latest/userguide/rotate-secrets_managed.html

## Breaking Changes
No.

## How Has This Been Tested?
- [x] I have updated at least one of the `examples/*` to demonstrate and validate my change(s)
- [x] I have tested and validated these changes using one or more of the provided `examples/*` projects

- [x] I have executed `pre-commit run -a` on my pull request

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -729,3 +729,37 @@ variable "engine_native_audit_fields_included" {
...
+}
+
+variable "master_user_password_rotation_duration" {
+  description = "The length of the rotation window in hours. For example, 3h for a three hour window."
+  type        = string
+  default     = null
+}
+
+variable "master_user_password_rotation_schedule_expression" {
+  description = "A cron() or rate() expression that defines the schedule for rotating your secret. Either automatically_after_days or schedule_expression must be specified."
```
Comment (bryantbiggs) [inline variables.tf:762]:
```suggestion
  description = "A cron() or rate() expression that defines the schedule for rotating your secret. Either `master_user_password_rotation_automatically_after_days` or `master_user_password_rotation_schedule_expression` must be specified"
```

File: variables.tf [hunk 2]
```diff
@@ -729,3 +729,37 @@ variable "engine_native_audit_fields_included" {
...
+}
+
+variable "master_user_password_rotate_immediately" {
+  description = "Specifies whether to rotate the secret immediately or wait until the next scheduled rotation window."
+  type        = bool
+  default     = null
+}
+
+variable "master_user_password_rotation_automatically_after_days" {
+  description = "Specifies the number of days between automatic scheduled rotations of the secret. Either automatically_after_days or schedule_expression must be specified."
```
Comment (bryantbiggs) [inline variables.tf:750]:
```suggestion
  description = "Specifies the number of days between automatic scheduled rotations of the secret. Either `master_user_password_rotation_automatically_after_days` or `master_user_password_rotation_schedule_expression` must be specified"
```

**Review by bryantbiggs — COMMENTED** [review]
looks great! just small description updates for the variables we're using here

**Review by bryantbiggs — APPROVED** [review]
Thank you!

**Comment by antonbabenko** [comment]
This PR is included in version 9.2.0 :tada:

**Pull request merged**